modules/ROOT/pages/6.7.2-release-notes.adoc

```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.
 
-// CCFR
+This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the origional function was not changed to consider the newly introduced context.
+
+As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the users developer console.
+
+{productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.
```

Comment:
@HAFRMO I agree, but im not sure if providing less info will be sufficient enough for a docs entry, as I dont believe telling the customer that we "copied over a function without concern of context" is a good idea either as mentioned in TINY-10263 docs outline? as this might give them the wrong impression?

Reply:
I think we just need to explain that an exception was thrown if the integrator called `getModifierState` on the event object that the editor provide though the `keydown` event handler call. How to explain that in simpler terms I'm not sure.

Reply:
I think Spocke's right on this one.

Reply:
```suggestion
{productname} 6.7.2 addresses this, now, when a integrator uses the `getModifierState` on the event object a keydown event no longer triggers an error.
```